web/components/admin/Toolbar/index.js

```diff
@@ -10,10 +10,17 @@ class Toolbar extends React.Component {
       mentor: { value: 'mentor', label: 'Mentor' },
       volunteer: { value: 'volunteer', label: 'Volunteer' },
     };
+    this.sortSelectLabels = {
+      name: { value: 'name', label: 'Name' },
+      email: { value: 'email', label: 'Email' },
+      score: { value: 'score', label: 'Score' },
+      timestamp: { value: 'timestamp', label: 'Timestamp' },
+    };
   }
 
   render() {
-    const { applicantType, switchApplicantType, className } = this.props;
+    const { applicantType, switchApplicantType, className,
+      switchSortType, sortType, reverseSort, reverse } = this.props;
     return (
       <div className={`toolbar fill-width ${className}`}>
```

Comment:
When in doubt add `flex` to the `className` of the container, fixes 99% of problems I swear 😛